handbook/engineering/campaigns/goals.md

```diff
@@ -1,66 +1,69 @@
 
-### Grow adoption of campaigns
 
-Once we’re out of beta, we want everyone to start using campaigns!
+## Design and build the most robust, yet easy to use solution for creating changesets across many repositories
 
-We need to follow-up with customers who are already using campaigns to make sure that we’re taking their input into consideration when prioritizing new features.
+Campaigns need to be easy to explain, easy to understand, easy to start using, and easy to explore. At the same time, it should be the most powerful tool available for making programatic changes to multiple repositories. 
 
-We need to make it as easy as possible for new customers to try out campaigns:
+How do we do this:
+  * Clean conceptual model
+  * Expressive syntax
+  * Understand our customers environments and requirements
+  * Strong alignment of features to developer objectives
+  * Excellent docs, tutorials, demos/training
+  * Best debugging experience
 
-- **Even smoother on-ramp:** Minimize the number of steps it takes a user to go from “found Campaigns in the Sourcegraph menu” to “has created a pull request, making a meaningful change”.
-- **Easy to try:** Allow users to run campaigns on sourcegraph.com to try them out (requires user-specific tokens).
-- **Examples:** Document as many meaningful example campaigns as possible.
-- **More examples:** Provide ready-to-go campaign specs that users can easily run.
+Our success metrics:
+  * Reduction of feature requests over time
 
-[Looker dashboard with usage metrics (internal only)](https://sourcegraph.looker.com/dashboards/136)
+## Build the best solution for managing large numbers of changesets
 
-### Make it easier for customers to change the code they want to change in their preferred way
+Much of the work in completing a large scale change across many repos is in getting the changesets merged. We want to be the best tool for all aspects of changeset management.
 
-As we learn how people use Campaigns, we also need to focus on _how they wish they could use_ Campaigns.
+How do we do this:
+  * Help users understand the overall status of their campaign
+  * Provide tools to find and resolve problematic changesets
+  * Provide flexible tools to prompt reviewers and owners to merge the created changesets in whatever way best suits the customer
+  * Integrate with issue tracking systems
 
-For example, Campaigns run on _repositories_ vs. on specific search results. When running a campaign, the code that’s being run to change code doesn’t have access to the specific search results (i.e. file + line number), only to the repository. That makes it hard to make really granular changes based on search results. (A possible solution would be a more refined interface between search results and user-supplied code. For example: we could pass lines of _<filename>:<lineno>_ on stdin to each command that’s executed in a repository.)
+Our success metrics:
+  * Seconds from changeset created to changeset merged
```

Comment:
Didn't we say in a previous call that this is bad metric because, ultimately, it's out of our hands? If an intern at a company blasts 500 repositories with changesets by accident and then they're all closed, not merged — what happens to our metrics?

Since we say "you bring your code to make the change" we don't have an influence on the changes that are being made, but I think the time-to-merge is _highly_ related to the change itself.

Reply:
I agree and yes we did decide that, but based on things we learned in the last month, this is a primary metric companies will gauge our value on. 

Perhaps the metric is something like: 'Percentage of published changesets merged' in campaigns older than 7 days is 60% or higher?